src/Jobs/Recruit.Vacancies.Jobs/ServiceCollectionExtensions.cs

```diff
@@ -39,17 +35,17 @@ public static void ConfigureJobServices(this IServiceCollection services, IConfi
                                                             });
 
             // Add Jobs
-            services.AddScoped<DomainEventsJob>();
-            services.AddScoped<ApprenticeshipProgrammesJob>();
-            services.AddScoped<VacancyStatusJob>();
-            services.AddScoped<EmployerDashboardGeneratorJob>();
-            services.AddScoped<PublishedVacanciesGeneratorJob>();
-            services.AddScoped<BankHolidayJob>();
-            services.AddScoped<QaDashboardJob>();
-            services.AddScoped<NonLevyAccountBlockerJob>();
-            services.AddScoped<VacancyAnalyticsSummaryGeneratorJob>();
+            services.AddScoped<DomainEventsQueueTrigger>();
+            services.AddScoped<UpdateApprenticeshipProgrammesQueueTrigger>();
+            services.AddScoped<VacancyStatusQueueTrigger>();
+            services.AddScoped<GenerateSingleEmployerDashboardQueueTrigger>();
+            services.AddScoped<GeneratePublishedVacanciesQueueTrigger>();
+            services.AddScoped<UpdateBankHolidayQueueTrigger>();
+            services.AddScoped<UpdateQaDashboardQueueTrigger>();
+            services.AddScoped<GenerateBlockedEmployersQueueTrigger>();
+            services.AddScoped<GenerateVacancyAnalyticsQueueTrigger>();
```

Comment:
Can you call this GenerateVacancyAnalyticsSummary and be aware that in prod this job is turned off through the disabledJobs config check where it will need to be updated.

Reply:
I'm not understanding why this particular queue trigger job is so special? It would be good if all queue triggers are called 'xxxxQueueTrigger'.  We can just add `GenerateVacancyAnalyticsQueueTrigger` to PROD config

Reply:
I meant could you call it `GenerateVacancyAnalyticsSummaryQueueTrigger`. The job isn't anything special but I would like for it to retain `Summary` in its title indicating it generates a summary document.

Reply:
👍